python/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ declare_mlir_python_sources(TorchMLIRPythonSources.Importers
   SOURCES
     extras/fx_importer.py
     extras/onnx_importer.py
+    fx.py
```

Comment:
I think we want a different group for this. Perhaps "PublicAPI". We're not ready to export this to downstreams yet, which is what this group does.

Reply:
okay created a PublicAPI group.